project/ScalazBuild.scala

```diff
@@ -27,7 +27,7 @@ object Scalaz {
     "-Ywarn-nullary-override",
     "-Ywarn-nullary-unit",
     "-Ywarn-numeric-widen",
-    "-Ywarn-unused:_,-imports",
+    "-Ywarn-unused:_,-imports,-locals",
```

Comment:
Why is this an issue?

Reply:
https://github.com/scala/bug/issues/10623

Reply:
thx, good to know :)

Reply:
I think using [`@silent` annotation](https://github.com/ghik/silencer) for that specific statement (with a `FIXME` that refer to the bug) would be better that disabling the warnings for the whole project.

Reply:
Fixed.